src/main/groovy/com/morpheusdata/scvmm/ScvmmProvisionProvider.groovy

```diff
@@ -2175,7 +2190,15 @@ class ScvmmProvisionProvider extends AbstractProvisionProvider implements Worklo
                 resizeRequest.volumesUpdate?.each { volumeUpdate ->
                     if (volumeUpdate.existingModel) {
                         //existing disk - resize it
+						def volumeCode = volumeUpdate.existing.type?.code ?: "standard"
```

Comment:
The hotresize code as found [here](https://github.com/gomorpheus/morpheus-ui/blob/7384008b06911795df8af6e43fa4757f8a72a645/clouds/scvmm/grails-app/services/com/morpheus/provision/ScvmmProvisionService.groovy#L480) couldnot be located in the ported code.